gradle/libs.versions.toml

```diff
@@ -11,6 +11,8 @@ kotlin-test = { module = "org.jetbrains.kotlin:kotlin-test", version.ref = "kotl
 kotlinx-coroutines-core = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version.ref = "kotlinx-coroutines" }
 kotlinx-coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-test", version.ref = "kotlinx-coroutines" }
 
+androidx-collection = "androidx.collection:collection:1.5.0-alpha01"
```

Comment:
I don't know why, when using version `1.4.3` (the latest stable) gradle sync closes with the error `transformNativeMainCInteropDependenciesMetadataForIde` for some modules.